CMakeLists.txt

```diff
@@ -618,6 +620,10 @@ if(WITH_OPTIM)
             endif()
         endif()
     elseif(BASEARCH_S360_FOUND)
+        if(WITH_CRC32_VX)
```

Comment:
We allow disabling individual optimizations on other platforms too... it's useful when comparing outputs of different implementations.

Reply:
The other platforms always include arch.c so long as (WITH_OPTIM=ON).

x86:
![image](https://user-images.githubusercontent.com/1364432/124640110-ce480900-de41-11eb-9ac9-39546cdecca6.png)

arm:
![image](https://user-images.githubusercontent.com/1364432/124640224-f9caf380-de41-11eb-87ab-c862cc13acb3.png)

power:
![image](https://user-images.githubusercontent.com/1364432/124640251-03ecf200-de42-11eb-9971-fedcbdf0b05b.png)

And actually I think power should be changed to always include power.c if `BASEARCH_PPC_FOUND AND WITH_OPTIM`.

Reply:
PowerPC optimizations currently only support Linux... I already separated power.h/c from Power8 intrinsics test when I added optimizations for earlier processors.

Reply:
I think I'll keep this part as is, since different architectures do this differently anyway. If at some point it becomes desirable to make this more uniform, it will be easy to change.